### PR TITLE
Log some broadcast client errors only as warnings

### PR DESCRIPTION
**Pull request by joshuacolvin0**

### CODE REVIEW

File: packages/arb-util/broadcastclient/broadcastclient.go [hunk 1]
```diff
@@ -160,6 +160,8 @@ func (bc *BroadcastClient) startBackgroundReader(ctx context.Context, messageRec
 				}
 				if strings.Contains(err.Error(), "i/o timeout") {
 					logger.Error().Str("feed", bc.websocketUrl).Msg("Server connection timed out without receiving data")
+				} else if strings.Contains(err.Error(), "EOF") {
```
Comment (PlasmaPower) [inline packages/arb-util/broadcastclient/broadcastclient.go:163]: Can we use `errors.Is(err, io.EOF) || errors.Is(err, io.UnexpectedEOF)`? That should be a bit more stable. This also applies to the other places where we check for specific errors.

Comment (joshuacolvin0) [reply packages/arb-util/broadcastclient/broadcastclient.go]: Sounds good.  All the other `strings.Contains(err` code is either checking errors from third party libraries that don't provide the error object for comparing, or the string is set in C++ code.

Comment (PlasmaPower) [reply packages/arb-util/broadcastclient/broadcastclient.go]: Ah, that's too bad. Thanks for changing this one though!

**Review by PlasmaPower — APPROVED** [review]
LGTM

**Pull request merged**